openedx_lti_tool_plugin/views.py

```diff
@@ -429,11 +430,14 @@ def handle_ags(
         if AGS_SCORE_SCOPE not in ags_endpoint.get('scope', []):
             raise LtiToolLaunchException(_(f'Missing required AGS scope: {AGS_SCORE_SCOPE}'))
 
-        return LtiGradedResource.objects.get_or_create(
-            lti_profile=lti_profile,
-            context_key=context_key,
-            lineitem=lineitem,
-        )
+        try:
+            LtiGradedResource.objects.get_or_create(
+                lti_profile=lti_profile,
+                context_key=context_key,
+                lineitem=lineitem,
+            )
+        except ValidationError as exc:
+            raise LtiToolLaunchException(_(exc.messages[0])) from exc
```

Comment:
Why is it necessary to index 0? @alexjmpb

Reply:
@Squirrel18 Because it is validating the fields with full clean, the attribute messages returns a list with the messages for all fields.